src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/GlobalFlowStateAnalysis/GlobalFlowStateAnalysisValueSet.cs

```diff
@@ -40,17 +40,49 @@ internal sealed class GlobalFlowStateAnalysisValueSet : CacheBasedEquatable<Glob
             Kind = kind;
         }
 
-        public GlobalFlowStateAnalysisValueSet(IAbstractAnalysisValue analysisValue)
-            : this(ImmutableHashSet.Create(analysisValue), ImmutableHashSet<GlobalFlowStateAnalysisValueSet>.Empty, height: 0, GlobalFlowStateAnalysisValueSetKind.Known)
+        public static GlobalFlowStateAnalysisValueSet Create(
+            ImmutableHashSet<IAbstractAnalysisValue> analysisValues,
+            ImmutableHashSet<GlobalFlowStateAnalysisValueSet> parents,
```

Comment:
😕 Why do we have `CreateWithParents`, and also have `Create` with `parents`?

Reply:
Ah, I wanted to rename `CreateWithParents` to `Create` but forgot to do that change, I can rename.